### ISSUE TITLE
media_player.select_source in HA triggers start of pauzed queue in MAss

### ISSUE THREAD

**Issue by bollewolle** [music-assistant/hass-music-assistant#1939]
### What version of Music Assistant has the issue?

2.0.0b102

### The problem

When in Music Assistant a queue is paused on my MA player "Office Small (DLNA)", and when I then do in Home Assistant trigger the service "media_player.select_source" on the Home Assistant entity "media_player.office_small_marantz_nd8006" to start a Tune-In radio station directly on my Heos device, than it looks like in Music Assistant it starts playing the pauzed queue again on my MA player "Office Small (DLNA)".

Notes:
- MA player "Office Small (DLNA)" is the DLNA player of my HEOS capabable Marantz ND8006 network streamer
- Home Assistant entity "media_player.office_small_marantz_nd8006" is the entity created by the Heos integration in Home Assistant

The problem is that I think it might be specific to Heos devices, but could be wrong of course. I don't have another player at hand to validate this though.

### How to reproduce

1. Go to Music Assistant and start playing an album on a player that is NOT coming from Home Assistant directly (in my example the DNLA player of a Heos device)
2. Pauze the music
3. Go to Home Assistant (best to do this in a new window) --> Developer Tools --> Services and select as service "Media Player: Select source". 
4. Choose as entity the Home Assistant specific media_player of the same device (in my example the one created by my Heos integration in Home Assistant) and as Source fill in one of the sources that this media player supports (you might need to look this up in your specific instance)
5. Keep an eye on the Music Assistant pauzed player and trigger the "Call Service"
6. At this moment it seems that the Music Assistant pauze queue starts playing again.

### Additional information

A screencapture can be found on this Discord post: https://discord.com/channels/753947050995089438/1083643113484070953/1209185172067713084

### What version of Home Assistant Core are your running

2024.2.2

### What type of installation are you running?

Home Assistant OS

### On what type of hardware are you running?

Alternative

**Comment by marcelveldt** [comment]
Can you please re-test this with b102 ? I identified a few places that may induce this

**Comment by bollewolle** [comment]
I'm currently on 2.0.0b103 and checked it but it seems it still happens. Is there a log or something I could provide to fine-tune into the possible cause?

**Comment by bollewolle** [comment]
Perhaps an addition: I tried disabling all the Player Providers in Music Assistant that are discoverd by Music Assistant (e.g. via DLNA or Airplay) and then used only the Home Assistant ones that are exposed to MA. I get similar behaviour there as well.

I made a screenrecording which might explain better what I see:
https://github.com/music-assistant/hass-music-assistant/assets/368306/4e2755ce-0ea4-4563-9016-1b777a2e26c8

So you see that MA is playing in the beginning, I pauze it, then go to the same media_player in HA and select one of the source_list dropdown. At that point it looks as like the the player in MA is starting to play again. 

What's also weird is that once the Source is playing for a few minutes, it seems that MA is forcing itself again on the media player, disconnects the source playing and just starts playing its queue again. It also gets very weird because at that point what I hear playing doesn't correspond anymore with what is supposed to be playing according to MA.

I only have Heos devices that have the "source_list" functionality, but a thought I had was that perhaps this could be tested on other devices that support the "source_list" attribute in HA as well? I believe Sonos uses it as well for radio stations or favourites set on the Sonos device?

Do let me know if there's more to test. I can already state that I did not trigger anything in the logs of MA or HA.

PS: also, but perhaps a known issue and perhaps unrelated to this current ticket, I just see "Url Stream - Url Stream" as artist and song in HA and no album art.
